io.ossia.score.yml

```diff
@@ -0,0 +1,333 @@
+id: io.ossia.score
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: ossia-score
+finish-args:
+  # BLE support
+  - --allow=bluetooth
+
+  # Access user's data and media libraries, plug-ins etc
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission.

Reply:
hmm, users of the software generally have various data hard drives and folders mounted with audio, video content, etc. what would be the most frictionless way to handle this compared to the AppImage ? (every permission dialog generally ends up loosing less technical users from the UX testing I could run over the years)